package.json

```diff
@@ -74,13 +74,9 @@
     "lint": "npm run flow:check && npm run linter:check",
     "cy:open": "d2-utils-cypress open --appStart 'yarn start:forCypress'",
     "cy:run": "d2-utils-cypress run --appStart 'yarn start:forCypress'",
-    "verifyCacheVersion": "node scripts/verifyCacheVersion.js"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "d2-app-scripts i18n extract && git add ./i18n/",
-      "pre-push": "yarn linter:check && yarn flow:check"
-    }
+    "verifyCacheVersion": "node scripts/verifyCacheVersion.js",
+    "prepare": "husky install",
+    "i18n:add": "d2-app-scripts i18n extract && git add ./i18n/"
```

Comment:
Do you think it makes sense to add: `"postinstall": "yarn prepare"` to ensure husky is installed when doing `yarn (install)`?

Reply:
Hello,
From what I understood it will be automatically run as part of the yarn scripts lifecycle every time after your packages are installed (when you run yarn). It is what the Husky documentation recommends https://typicode.github.io/husky/#/?id=install. Therefore we don’t have to run anything manually.

Reply:
Thanks, did a second test and it seems to work 🤔

Reply:
After all, I use the `postinstall` step you suggested because it has support in both `yarn` versions. Thank you!